Shopify/ruby-lsp
# Index core classes using RBS

Step towards https://github.com/Shopify/ruby-lsp/issues/1335

To test:

- Disable the Sorbet extension and hover over a core class name, e.g. String.
- Verify you see the docs in the hover
- Use Go To Definition and verify it takes you to the RBS file with the String definitions.

After this we can move on to methods.
